src/controllers/main/main.ts

```diff
@@ -1409,14 +1412,18 @@ export class MainController extends EventEmitter {
           // TODO: Implement a way to choose the key type to broadcast with.
           feePayerKeys.find((key) => key.type === accountOp.signingKeyType) || feePayerKeys[0]
         if (!feePayerKey) {
-          return this.#throwAccountOpBroadcastError(
+          return await this.#throwBroadcastAccountOp(
             new Error(
-              `Key with address: ${accountOp.gasFeePayment!.paidBy} for account with address: ${
-                accountOp.accountAddr
-              } not found`
+              `Key with address ${shortenAddress(
+                accountOp.gasFeePayment!.paidBy,
+                13
+              )} for account with address ${shortenAddress(accountOp.accountAddr, 13)} not found.`
```

Comment:
`#throwBroadcastAccountOp` is used ~13 times, 12 of which initialize a new Error(). Imo this can be simplified by making `#throwBroadcastAccountOp` accept an object as a param with 2 keys:
```
error?: Error
message?: string
```
, then the 12 calls can be simplified to `this.#throwBroadcastAccountOp({ message: 'something' })`

and then in the method:
```
let message = message || 'Unable to broadcast the transaction.'
...
return Promise.reject(new EmittableError({ level: 'major', message, error || new Error(message) }))
```

Reply:
Yes, I see your point. Refactored in https://github.com/AmbireTech/ambire-common/pull/861/commits/542b462ab6abb324aaefba3661b0c3dd99f509aa